modules/docextract/lib/refextract_kbs.py

```diff
@@ -194,7 +193,6 @@ def institute_num_pattern_to_regex(pattern):
     """
     simple_replacements = [
                             ('9',    r'\d'),
-                            ('9+',   r'\d+'),
                             ('w+',   r'\w+'),
```

Comment:
It is currently unused, but I didn't want to remove it as it might be useful in the future.